readthedocs/proxito/views/mixins.py

```diff
@@ -68,6 +71,32 @@ def _serve_docs(
             download=download,
         )
 
+    def _trak_pageview(self, project, path, request):
+        """Create an audit log of the page view if audit is enabled."""
+        # Remove any query args (like the token access from AWS).
+        path_only = urlparse(path).path
+        track_file = any(
+            path_only.endswith(ext)
+            for ext in ['.html', '.pdf', '.epub', '.zip']
```

Comment:
I think `endswith` accepts a list to look for: https://docs.python.org/3/library/stdtypes.html#str.endswith

Reply:
Nice!